css/joomla.css

```diff
@@ -371,6 +371,11 @@ div#toolbar-box, div#toolbar-box div.m{
   display: block;
 }
 
+/* dev/core#874 the width:auto styling above causes the menubar colour picker to be squeezed */
+.crm-container input.crm-form-color {
+  width: 3.6em;
```

Comment:
Would this work?

```suggestion
  width: initial;
```

Reply:
@colemanw tried that and doesn't appear to work